src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.civil.controllers.cases;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.civil.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.CoreCaseDataService;
+
+import java.math.BigDecimal;
+
+@Api
+@Slf4j
+@RestController
+@AllArgsConstructor
+@RequestMapping(
+    path = "/cases",
+    produces = MediaType.APPLICATION_JSON_VALUE
+)
+public class CasesController {
+
+    private final CoreCaseDataService coreCaseDataService;
+    private final CaseDetailsConverter caseDetailsConverter;
+
+    @GetMapping(path = {
+        "/{cid}",
+    })
+    @ApiOperation("get case by id from CCD")
+
+    public ResponseEntity<CaseData> getClaimById(
+        @PathVariable("cid") Long claimId
+    ) {
+        log.info(
+            "Received CaseId: {}",
+            claimId
+        );
+
+        var caseDataResponse = caseDetailsConverter.toCaseData(coreCaseDataService.getCase(claimId).getData());
+        caseDataResponse.setTotalClaimAmount(BigDecimal.valueOf(150.00));
```

Comment:
Why do we want to set this value as 150? Is it just for testing purposes?

Reply:
because in the ccd's call this field is null I don't know why this field is in the data base.

Reply:
I figured out that this field wasn't show us because was a correct role in ccd-definition, I removed this line.